packages/forms/src/Form.test.js

```diff
@@ -142,6 +144,13 @@ describe('<Form/>', () => {
 		expect(Form.displayName).toBe('TalendForm');
 	});
 
+	it('should render uiform', () => {
+		const props = initProps();
+		// when
+		wrapper = shallow(<Form data={dataUIForm} {...props} />);
+		expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
The shallow snapshot doesn't check that we have a UIForm.
1. Don't forget to check the snapshot when you write/modify one
2. Here you can check that without snapshot (which will log a lot more info than what you want to test)

Reply:
My bad I will learn to read